src/app/public/modules/needs-attention/types/action-hub-needs-attention.ts

```diff
@@ -0,0 +1,13 @@
+import { NavigationExtras } from '@angular/router';
+
+export type SkyActionHubNeedsAttention = {
+  message?: string;
+  permalink?: {
+    route?: {
+      commands: any[];
+      extras: NavigationExtras;
```

Comment:
Should `extras` be optional here? I recently updated the action button permalink type to make it optional since Angular strict mode caught the places where I wasn't specifying it.

Reply:
@johnhwhite - it appears you changed this in the `link-list/types/link.ts` but not here. Does it need updated here too?